### PR TITLE
NeoToken: add NEP-27 to supported standards list starting from Echidna

### PR DESCRIPTION
**Pull request by AnnaShaleva**
# Description

https://github.com/neo-project/neo/pull/3597 introduces `onNEP17Payment` handler to native NeoToken contract starting from Echidna hardfork. We need to update the list of supported standards respectively.

## Type of change

- [ ] Optimization (the change is only an optimization)
- [ ] Style (the change is only a code style for better maintenance or standard purpose)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] `TestGenesisNativeState` unit-test.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Neo/SmartContract/Native/FungibleToken.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ protected FungibleToken() : base()
             Factor = BigInteger.Pow(10, Decimals);
         }
 
-        protected override void OnManifestCompose(ContractManifest manifest)
+        protected override void OnManifestCompose(IsHardforkEnabledDelegate hfChecker, uint blockHeight, ContractManifest manifest)
```
Comment (roman-khimov) [inline src/Neo/SmartContract/Native/FungibleToken.cs:68]: Yeah, I knew it's non-trivial to get HF-dependent standards.

**Review by roman-khimov — COMMENTED** [review]
Should be OK, technically it's correct to have NEP-27 in standards.

**Comment by AnnaShaleva** [comment]
Tests are failing, but due to another reason:
```
 expect: read eof
expect: set expect_out(spawn_id) "exp3"
expect: set expect_out(buffer) "System.ArgumentException: Invalid multisig parameters: m=0, publicKeys.Count=0\r\n   at Neo.SmartContract.Contract.CreateMultiSigRedeemScript(Int32 m, IReadOnlyCollection`1 publicKeys) in /home/runner/work/neo/neo/src/Neo/SmartContract/Contract.cs:line 106\r\n   at Neo.SmartContract.Contract.GetBFTAddress(IReadOnlyCollection`1 pubkeys) in /home/runner/work/neo/neo/src/Neo/SmartContract/Contract.cs:line 152\r\n   at Neo.NeoSystem.CreateGenesisBlock(ProtocolSettings settings) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 153\r\n   at Neo.NeoSystem..ctor(ProtocolSettings settings, IStoreProvider storageProvider, String storagePath) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 135\r\n   at Neo.NeoSystem..ctor(ProtocolSettings settings, String storageProvider, String storagePath) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 121\r\n   at Neo.CLI.MainService.Start(CommandLineOptions options) in /home/runner/work/neo/neo/src/Neo.CLI/CLI/MainService.cs:line 381\r\n   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_1(Object state)\r\n   at System.Threading.ThreadPoolWorkQueue.Dispatch()\r\n   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()\r\ncreate wallet ./bin/Neo.CLI/test-wallet1.json\r\n"
expect: spawn id exp3 not open
    while executing
"expect {
    "password:" { send "asd\n" }
    "error" { exit 2 }
    timeout { exit 1 }
}"
```

**Comment by Jim8y** [comment]
> Tests are failing, but not due to this PR:
> 
> ```
>  expect: read eof
> expect: set expect_out(spawn_id) "exp3"
> expect: set expect_out(buffer) "System.ArgumentException: Invalid multisig parameters: m=0, publicKeys.Count=0\r\n   at Neo.SmartContract.Contract.CreateMultiSigRedeemScript(Int32 m, IReadOnlyCollection`1 publicKeys) in /home/runner/work/neo/neo/src/Neo/SmartContract/Contract.cs:line 106\r\n   at Neo.SmartContract.Contract.GetBFTAddress(IReadOnlyCollection`1 pubkeys) in /home/runner/work/neo/neo/src/Neo/SmartContract/Contract.cs:line 152\r\n   at Neo.NeoSystem.CreateGenesisBlock(ProtocolSettings settings) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 153\r\n   at Neo.NeoSystem..ctor(ProtocolSettings settings, IStoreProvider storageProvider, String storagePath) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 135\r\n   at Neo.NeoSystem..ctor(ProtocolSettings settings, String storageProvider, String storagePath) in /home/runner/work/neo/neo/src/Neo/NeoSystem.cs:line 121\r\n   at Neo.CLI.MainService.Start(CommandLineOptions options) in /home/runner/work/neo/neo/src/Neo.CLI/CLI/MainService.cs:line 381\r\n   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_1(Object state)\r\n   at System.Threading.ThreadPoolWorkQueue.Dispatch()\r\n   at System.Threading.PortableThreadPool.WorkerThread.WorkerThreadStart()\r\ncreate wallet ./bin/Neo.CLI/test-wallet1.json\r\n"
> expect: spawn id exp3 not open
>     while executing
> "expect {
>     "password:" { send "asd\n" }
>     "error" { exit 2 }
>     timeout { exit 1 }
> }"
> ```

Test fails only in the workflow, i could not reproduce it locally,,, thus i can not debug it....

**Review by Jim8y — APPROVED** [review]

**Review by roman-khimov — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Comment by shargon** [comment]
UT problem could be fixed in a different PR

**Pull request merged**